src/main/java/ch/njol/skript/expressions/ExprNow.java

```
@@ -15,24 +15,56 @@
 import ch.njol.skript.util.Date;
 import ch.njol.util.Kleenean;
 
+import java.time.DateTimeException;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
 @Name("Now")
 @Description("The current <a href='classes.html#date'>system time</a> of the server. Use <a href='#ExprTime'>time</a> to get the <a href='classes.html#time'>Minecraft time</a> of a world.")
 @Examples({"broadcast \"Current server time: %now%\""})
-@Since("1.4")
+@Since("1.4, INSERT VERSION (timezones)")
 public class ExprNow extends SimpleExpression<Date> {
 	
 	static {
-		Skript.registerExpression(ExprNow.class, Date.class, ExpressionType.SIMPLE, "now");
+		Skript.registerExpression(ExprNow.class, Date.class, ExpressionType.SIMPLE, "now [timezone:in time[ ]zone %-string%]");
 	}
+
+	private boolean usingTimezone;
+	private Expression<String> timezone;
 	
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		usingTimezone = parseResult.hasTag("timezone");
+		timezone = (Expression<String>) exprs[0];
 		return true;
 	}
 	
 	@Override
-	protected Date[] get(final Event e) {
-		return new Date[] {new Date()};
+	protected Date[] get(Event event) {
+		if (usingTimezone) {
+			String timezone = this.timezone.getSingle(event);
+			if (timezone == null) {
+				return new Date[0];
```

Should probably throw a runtime error here